diff -ru pristine/wom/enums.py /store/droptracker/disc/venv/lib/python3.11/site-packages/wom/enums.py
--- pristine/wom/enums.py	1980-01-01 00:00:00.000000000 +0000
+++ /store/droptracker/disc/venv/lib/python3.11/site-packages/wom/enums.py	2026-07-02 19:13:59.892189370 +0000
@@ -103,6 +103,7 @@
     Herblore = "herblore"
     Agility = "agility"
     Thieving = "thieving"
+    Sailing = "sailing"
     Slayer = "slayer"
     Farming = "farming"
     Runecrafting = "runecrafting"
@@ -125,13 +126,16 @@
     PvpArena = "pvp_arena"
     SoulWarsZeal = "soul_wars_zeal"
     GuardiansOfTheRift = "guardians_of_the_rift"
-
+    CollectionsLogged = "collections_logged"
+    
     # Bosses
     AbyssalSire = "abyssal_sire"
     AlchemicalHydra = "alchemical_hydra"
+    Amoxliatl = "amoxliatl"
     Araxxor = "araxxor"
     Artio = "artio"
     BarrowsChests = "barrows_chests"
+    Brutus = "brutus"
     Bryophyta = "bryophyta"
     Callisto = "callisto"
     Calvarion = "calvarion"
@@ -147,11 +151,13 @@
     DagannothRex = "dagannoth_rex"
     DagannothSupreme = "dagannoth_supreme"
     DerangedArchaeologist = "deranged_archaeologist"
+    DoomOfMokhaiotl = "doom_of_mokhaiotl"
     DukeSucellus = "duke_sucellus"
     GeneralGraardor = "general_graardor"
     GiantMole = "giant_mole"
     GrotesqueGuardians = "grotesque_guardians"
     Hespori = "hespori"
+    Hueycoatl = "the_hueycoatl"
     KalphiteQueen = "kalphite_queen"
     KingBlackDragon = "king_black_dragon"
     Kraken = "kraken"
@@ -162,6 +168,7 @@
     Nex = "nex"
     Nightmare = "nightmare"
     PhosanisNightmare = "phosanis_nightmare"
+    MaggotKing = "maggot_king"
     Obor = "obor"
     PhantomMuspah = "phantom_muspah"
     Sarachnis = "sarachnis"
@@ -169,12 +176,14 @@
     Scurrius = "scurrius"
     Skotizo = "skotizo"
     SolHeredit = "sol_heredit"
+    ShellbaneGryphon = "shellbane_gryphon"
     Spindel = "spindel"
     Tempoross = "tempoross"
     TheGauntlet = "the_gauntlet"
     TheCorruptedGauntlet = "the_corrupted_gauntlet"
     TheLeviathan = "the_leviathan"
     TheWhisperer = "the_whisperer"
+    TheRoyalTitans = "the_royal_titans"
     TheatreOfBlood = "theatre_of_blood"
     TheatreOfBloodHard = "theatre_of_blood_hard_mode"
     ThermonuclearSmokeDevil = "thermonuclear_smoke_devil"
@@ -189,7 +198,7 @@
     Wintertodt = "wintertodt"
     Zalcano = "zalcano"
     Zulrah = "zulrah"
-
+    Yama = "yama"
     # Computed Metrics
     Ehp = "ehp"
     Ehb = "ehb"
@@ -219,6 +228,7 @@
         Metric.Herblore,
         Metric.Agility,
         Metric.Thieving,
+        Metric.Sailing,
         Metric.Slayer,
         Metric.Farming,
         Metric.Runecrafting,
@@ -245,6 +255,7 @@
         Metric.PvpArena,
         Metric.SoulWarsZeal,
         Metric.GuardiansOfTheRift,
+        Metric.CollectionsLogged
     }
 )
 """Set containing activities."""
@@ -253,9 +264,11 @@
     {
         Metric.AbyssalSire,
         Metric.AlchemicalHydra,
+        Metric.Amoxliatl,
         Metric.Araxxor,
         Metric.Artio,
         Metric.BarrowsChests,
+        Metric.Brutus,
         Metric.Bryophyta,
         Metric.Callisto,
         Metric.Calvarion,
@@ -271,17 +284,20 @@
         Metric.DagannothRex,
         Metric.DagannothSupreme,
         Metric.DerangedArchaeologist,
+        Metric.DoomOfMokhaiotl,
         Metric.DukeSucellus,
         Metric.GeneralGraardor,
         Metric.GiantMole,
         Metric.GrotesqueGuardians,
         Metric.Hespori,
+        Metric.Hueycoatl,
         Metric.KalphiteQueen,
         Metric.KingBlackDragon,
         Metric.Kraken,
         Metric.Kreearra,
         Metric.KrilTsutsaroth,
         Metric.LunarChests,
+        Metric.MaggotKing,
         Metric.Mimic,
         Metric.Nex,
         Metric.Nightmare,
@@ -292,6 +308,7 @@
         Metric.Scorpia,
         Metric.Scurrius,
         Metric.Skotizo,
+        Metric.ShellbaneGryphon,
         Metric.SolHeredit,
         Metric.Spindel,
         Metric.Tempoross,
@@ -299,6 +316,7 @@
         Metric.TheCorruptedGauntlet,
         Metric.TheLeviathan,
         Metric.TheWhisperer,
+        Metric.TheRoyalTitans,
         Metric.TheatreOfBlood,
         Metric.TheatreOfBloodHard,
         Metric.ThermonuclearSmokeDevil,
@@ -313,6 +331,7 @@
         Metric.Wintertodt,
         Metric.Zalcano,
         Metric.Zulrah,
+        Metric.Yama
     }
 )
-"""Set containing bosses."""
+"""Set containing bosses."""
\ No newline at end of file
diff -ru pristine/wom/models/players/enums.py /store/droptracker/disc/venv/lib/python3.11/site-packages/wom/models/players/enums.py
--- pristine/wom/models/players/enums.py	1980-01-01 00:00:00.000000000 +0000
+++ /store/droptracker/disc/venv/lib/python3.11/site-packages/wom/models/players/enums.py	2025-03-08 05:00:00.000000000 +0000
@@ -327,3 +327,5 @@
     Za = "ZA"
     Zm = "ZM"
     Zw = "ZW"
+    GB_Sct = "GB_SCT"
+    GB_Wls = "GB_WLS"
